### PR TITLE
Getting started: add Fly.io Essentials doc

### PR DESCRIPTION
**Pull request by andie787**
### Summary of changes

Add a new doc that is a primer on some Fly.io basics like what are Machines and what is Fly Launch?

Add a glossary of Fly.io terms. We can add to this (6PN, Flycast, etc.)

### Preview

https://aa-docs-3.fly.dev/docs/getting-started/essentials/

### CODE REVIEW

File: getting-started/essentials.html.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- Perform coordinated deployment of Machines for your app with the `fly deploy` command.
+- Scale your app's Machines[ horizontally](/docs/apps/scale-count/) or[ vertically](/docs/apps/scale-machine/) with the `fly scale` command.
+
+Again, Fly Launch is built on Machines: you can use Fly Launch to manage the scale of your application with a single command, or you can interact directly with Machines for fine-grained control.
+
+Learn more about [Fly Launch](/docs/apps/).
+
+## Fly Apps, Fly Machines, and Fly Launch
+
+A Fly App is just a way to group Machines on the Fly Platform. When we talk about "your app" in how-to docs, we're talking about a Fly App.
```
Comment (catflydotio) [inline getting-started/essentials.html.md:39]: Here's a Fly Platform. Should it be "the Fly.io platform"?

File: getting-started/essentials.html.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+[**Fly Launch**](/docs/apps/): Our orchestrator that includes some really good stuff for app hosting, like the `fly launch` command to get started, `fly.toml` for configuration, the `fly deploy` command to deploy all your app's Machines into new versions/releases, and the `fly scale` command to scale Machines.
+
+[**Fly Machines**](/docs/machines/): [Firecracker microVMs]([https://firecracker-microvm.github.io/](https://firecracker-microvm.github.io/)) that launch quickly in any [region supported by Fly.io](/docs/reference/regions/). A VM, or virtual machine, functions like a physical computer, but is software-based. Multiple VMs can run, completely isolated, on one physical host.
+
+If you've deployed an app on the Fly Platform, then it's running on Fly Machines. There’s a fast [REST API ](/docs/machines/api/) to manage Machines, but you don't have to use the API. Use flyctl–the Fly CLI–to manage everything from the command line. And then there’s Fly Launch, which combines flyctl commands with a shared config to manage your app’s Machines as a group.
```
Comment (catflydotio) [inline getting-started/essentials.html.md:68]:
```suggestion
If you've deployed an app on Fly.io, then it's running on Fly Machines. There’s a fast [REST API ](/docs/machines/api/) to manage Machines, but you don't have to use the API. Use flyctl–the Fly CLI–to manage everything from the command line. And then there’s Fly Launch, which combines flyctl commands with a shared config to manage your app’s Machines as a group.
```

File: getting-started/essentials.html.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+## Fly Apps, Fly Machines, and Fly Launch
+
+A Fly App is just a way to group Machines on the Fly Platform. When we talk about "your app" in how-to docs, we're talking about a Fly App.
+
+You can manage a Fly App's Machines as a group with Fly Launch features. or you can run and manage individual Machines with the Machines API or with `fly machine` flyctl commands. You can even do all of these things in one Fly App. We’re not here to tell you what to build!
+
+## Fly.io glossary
+
+**The Fly.io platform**: Everything Fly.io.
```
Comment (catflydotio) [inline getting-started/essentials.html.md:45]: I'm not sure this needs to be defined, or if it does, a useful definition is probably more detailed?

**Review by catflydotio — APPROVED** [review]

File: getting-started/essentials.html.md (hunk 3 above)
Comment (andie787) [reply getting-started/essentials.html.md:45]: good point!

Comment (andie787) [reply getting-started/essentials.html.md]: Added a bit more to the definition.

**Pull request merged**